docs/_newsfragments/2213.newandimproved.rst

```diff
@@ -1,6 +1,6 @@
-Added kwarg ``partitioned`` to :meth:`~falcon.Response.set_cookie`
-to opt a cookie into partitioned storage, with a separate cookie jar per
-top-level site.
-See also
+A new keyword argument, `partitioned`, was added to
+:meth:`~falcon.Response.set_cookie` to opt a cookie into partitioned storage,
+with a separate cookie jar per each top-level site.
+(See also
 `CHIPS <https://developer.mozilla.org/en-US/docs/Web/Privacy/Privacy_sandbox/Partitioned_cookies>`__
-for a more detailed description of this experimental web technology.
+for a more detailed description of this experimental web technology.)
```

Comment:
```suggestion
for a more detailed description of this web technology.)
```

Reply:
Isn't it experimental any longer?

Reply:
not sure but I think it may get outdated if we leave experimental there. at the end of the day it's supported by chrome, so it's likely that it will become mainstream

Reply:
Makes sense. I applied your suggestion for brevity regardless of the actual status, it just reads cleaner without.